kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/ImageServiceIncludedKieBaseIntegrationTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.integrationtests.jbpm;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.kie.server.api.model.KieServerConfigItem;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.client.KieServicesException;
+import org.kie.server.integrationtests.shared.KieServerDeployer;
+
+import static org.junit.Assert.*;
+
+public class ImageServiceIncludedKieBaseIntegrationTest extends JbpmKieServerBaseIntegrationTest {
+
+    private static ReleaseId releaseId = new ReleaseId("org.kie.server.testing", "top-level-project",
+            "1.0.0.Final");
+
+    private static final String CONTAINER_ID = "top-level";
+    private static final String HIRING_PROCESS_ID = "hiring";
+    private static final String HIRING_2_PROCESS_ID = "hiring2";
+    private static final String HIRING_3_PROCESS_ID = "hiring3";
+
+    @BeforeClass
+    public static void buildAndDeployArtifacts() {
+
+        KieServerDeployer.buildAndDeployCommonMavenParent();
+        KieServerDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/definition-project").getFile());
+        KieServerDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/top-level-project").getFile());
+
+        createContainer(CONTAINER_ID, releaseId, new KieServerConfigItem("KBase", "customKB", ""));
+    }
+
+    @Test
+    public void testGetProcessImageViaUIClientTest() throws Exception {
```

Comment:
It seems this method does similar thing 3 times - if a new method, e.g. "assertImageViaUIClient" were introduced, it would prevent code duplication.